### PR TITLE
tools/dm: add the doc of using DM to replicate data

### PR DESCRIPTION
**Pull request by CaitinChen**
English doc: https://github.com/pingcap/docs/blob/master/tools/dm/practice.md
Related PR: https://github.com/pingcap/docs/pull/915

@lilin90 @csuzhangxc @GregoryIan  PTAL

This doc contains three links pointing to external docs:

- "使用 DM-Ansible 部署 DM" in deployment.md
- "使用 dmctl 加密上游 MySQL 用户密码" in deployment.md
- "上游 MySQL 实例配置前置检查" in manage-task.md (removed)

The doc is added in README: https://github.com/pingcap/docs-cn/pull/1147

### CODE REVIEW

File: tools/dm/practice.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  password: ""
+
+# 当前数据同步任务需要的全部上游 MySQL 实例配置。
+mysql-instances:
+-
+  # 上游实例或者复制组 ID，参考 `inventory.ini` 的 `source_id` 或者 `dm-master.toml` 的 `source-id 配置`。
+  source-id: "mysql-replica-01"
+  # 需要同步的库名或表名的黑白名单的配置项名称，用于引用全局的黑白名单配置，全局配置见下面的 `black-white-list` 的配置。
+  black-white-list: "global"
+  # Mydumper 的配置项名称，用于引用全局的 mydumper 配置。
```
Comment (amyangfei) [inline tools/dm/practice.md]:
```suggestion
  # mydumper 的配置项名称，用于引用全局的 mydumper 配置。
```
should `mydumper` keeps the official capitalization. I notice in English doc it is used as `mydumper` at the beginning of sentence.

File: tools/dm/practice.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  mydumper-config-name: "global"
+
+# 黑白名单全局配置，各实例通过配置项名引用。
+black-white-list:
+  global:
+    do-tables:                        # 需要同步的上游表的白名单。
+    - db-name: "test_db"              # 需要同步的表的库名。
+      tbl-name: "test_table"          # 需要同步的表的名称。
+
+# Mydumper 全局配置，各实例通过配置项名引用。
```
Comment (amyangfei) [inline tools/dm/practice.md]: ditto

File: tools/dm/practice.md (hunk 1 above)
Comment (CaitinChen) [reply tools/dm/practice.md]: @amyangfei 
I find that when "mydumper" is at the beginning of the sentence, sometimes it is "mydumper" while sometimes it is "Mydumper". OK. Let's use "mydumper" consistently.

File: tools/dm/practice.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    [[deploy]]
+    source-id = "mysql-replica-01"
+    dm-worker = "172.16.10.72:8262"
+
+    [[deploy]]
+    source-id = "mysql-replica-02"
+    dm-worker = "172.16.10.73:8262"
+    ```
+
+    > **注意**：`{ansible deploy}/conf/dm-master.toml` 中的 `{ansible deploy}` 表示 DM-Ansible 部署的目录。该目录使用 `deploy_dir` 参数配置。
```
[the remark this answers is not in the document]
Comment (GregoryIan) [reply tools/dm/practice.md]: ping @CaitinChen

**Comment by lilin90** [comment]
@CaitinChen Please address the comments. Thanks!

**Comment by CaitinChen** [comment]
@lilin90 @amyangfei PTAL again

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: tools/dm/practice.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+category: tools
+---
+
+# 使用 DM 同步数据
+
+本文介绍如何使用 DM (Data Migration) 同步数据。
+
+## 第 1 步：部署 DM 集群
+
+目前推荐使用 DM-Ansible 部署 DM 集群，具体部署方法参照 [使用 DM-Ansible 部署 DM](/tools/dm/deployment.md)。
```
Comment (lilin90) [inline tools/dm/practice.md]:
```suggestion
目前推荐使用 DM-Ansible 部署 DM 集群，具体部署方法参照 [使用 DM-Ansible 部署 DM 集群](/tools/dm/deployment.md)。
```

File: tools/dm/practice.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
+---
+title: 使用 DM 同步数据
```
Comment (lilin90) [inline tools/dm/practice.md]: Please add this file to README.md.

**Pull request merged**